support-frontend/assets/pages/subscriptions-redemption/api.js

```diff
@@ -27,7 +28,9 @@ function validate(userCode: string) {
   if (userCode === '') {
     return Promise.resolve({ valid: false, errorMessage: 'Please enter your code' });
   }
-  const validationUrl = `${getOrigin()}/subscribe/redeem/validate/${userCode}`;
+  const whyIsTestUser = getGlobal<string>('whyIsTestUser');
```

Comment:
wasn't sure if we should just grab the global constant from this part of the code or we should do it once centrally